docs(installation): added troubleshooting (#67)
bonitasoft/bonita-ici-doc: md/installation_guide.md

## Installation with a manually installed Elasticsearch

### Overview

In that case, you request the BICI installer to perform the same operations as in the full installation but it will not
try to start the BICI Storage Docker container and it will use the instance specified by configuration.

![Full installation using the installer](images/bici_installation_installer_no_elasticsearch.svg)

### BICI Installer Configuration

If you want to use a your own manually installed Elasticsearch, change the following properties in the `configuration.properties` file:
```
elasticsearch.port=<port of the elasticsearch>
elasticsearch.host=<host of the elasticsearch>
```

If you use the interactive installer mode, provide the host and port of the running Elasticsearch instance when requested
by the installer.

### Installation

Then run the installer like this (pass the configuration file option if needed):
```
bonita-ici startApp deploy 
```

## Troubleshooting

### Polling errors

When a polling execution fails, it can be seen in error in BICI Configuration Living Application.

> More information is available in the output logs on the BICI backend, either in the Standard output or in the log file.

Reviewer: ```suggestion
More information is available in the logs of the BICI Application Docker container.
```
